In [14]:
import pandas as pd
import numpy as np


In [22]:
data=pd.read_csv('BuyerRatio.csv')
data

Observed Values  East  West  North  South
0           Males    50   142    131     70
1         Females   435  1523   1356    750

In [17]:
Male=pd.Series(data.iloc[0,:])
Male

Observed Values    Males
East                  50
West                 142
North                131
South                 70
Name: 0, dtype: object

In [18]:
Female=pd.Series(data.iloc[1,:])
Female

Observed Values    Females
East                   435
West                  1523
North                 1356
South                  750
Name: 1, dtype: object

In [21]:
data

Observed Values  East  West  North  South
0           Males    50   142    131     70
1         Females   435  1523   1356    750

In [23]:
data.drop(["Observed Values"], axis =1, inplace=True)

In [24]:
data

East  West  North  South
0    50   142    131     70
1   435  1523   1356    750

In [25]:
import scipy.stats as stats
from scipy.stats import chi2_contingency

In [27]:
p_value=stats.chi2_contingency(data)

In [28]:
p_value

(1.595945538661058,
 0.6603094907091882,
 3,
 array([[  42.76531299,  146.81287862,  131.11756787,   72.30424052],
        [ 442.23468701, 1518.18712138, 1355.88243213,  747.69575948]]))

In [29]:
p_value[1]

0.6603094907091882

## P-Value (0.6603)> α (0.05), Therefore we can't reject the null Hypothesis.
Proportion of Male and Female across regions is same.